.github/workflows/docker.yml

```diff
@@ -0,0 +1,54 @@
+name: Push Docker images to DockerHub and ECR
+
+on:
+  push:
+    branches: [master]
```

Comment:
Suggest `[master,main]` (or whatever the syntax is) to future-proof. I think there might even be a `${{ primary_branch }}` or something now (I totally guessed at that name)

Reply:
It looks like I can't use any variables for the branch names, so it will be a combination.